### PR TITLE
Move bootstrap utility functons to the acctest package

### PR DESCRIPTION
**Pull request by zli82016**
1. Move functions in the file `bootstrap_iam_test_utils.go` to the acctest package
2. Move function in the file `bootsrap_test_utils.go` to the acctest package
3. Replace functions calls

If this PR is for Terraform, I acknowledge that I have:

- [ ] Searched through the [issue tracker](https://github.com/hashicorp/terraform-provider-google/issues) for an open issue that this either resolves or contributes to, commented on it to claim it, and written "fixes {url}" or "part of {url}" in this PR description. If there were no relevant open issues, I opened one and commented that I would like to work on it (not necessary for very small changes).
- [ ] Ensured that all new fields I added that can be set by a user appear in at least one [example](https://github.com/GoogleCloudPlatform/magic-modules/tree/main/mmv1/templates/terraform/examples) (for generated resources) or [third_party test](https://github.com/GoogleCloudPlatform/magic-modules/tree/main/mmv1/third_party/terraform/tests) (for handwritten resources or update tests).
- [ ] [Generated Terraform providers](https://github.com/GoogleCloudPlatform/magic-modules/blob/main/docs/content/docs/getting-started/generate-providers.md), and ran [`make test` and `make lint`](https://googlecloudplatform.github.io/magic-modules/docs/getting-started/run-provider-tests/#run-unit-tests) in the generated providers to ensure it passes unit and linter tests.
- [ ] [Ran](https://github.com/GoogleCloudPlatform/magic-modules/blob/main/docs/content/docs/getting-started/run-provider-tests.md) relevant acceptance tests using my own Google Cloud project and credentials (If the acceptance tests do not yet pass or you are unable to run them, please let your reviewer know).
- [ ] Read [Write release notes](https://googlecloudplatform.github.io/magic-modules/contribute/release-notes/) before writing my release note below.

**Release Note Template for Downstream PRs (will be copied)**

```release-note:none

```

### CODE REVIEW

**Comment by zli82016** [comment]
The test `TestAccFilestoreSnapshot_filestoreSnapshotFullExample` seems not related, as the file ` resource_filestore_snapshot_generated_test.go` with this test is not  in the list of changed files for this PR.
https://github.com/modular-magician/terraform-provider-google/compare/auto-pr-8244-old..auto-pr-8244

File: mmv1/third_party/terraform/tests/resource_firebase_extensions_instance_test.go.erb [hunk 1]
```diff
@@ -16,12 +16,12 @@ func TestAccFirebaseExtensionsInstance_firebaseExtentionsInstanceResizeImageUpda
 	context := map[string]interface{}{
 		"project_id":    acctest.GetTestProjectFromEnv(),
 		"location":      "us-central1",
-		"random_suffix": RandString(t, 10),
+		"random_suffix": acctest.RandString(t, 10),
```
Comment (roaks3) [inline mmv1/third_party/terraform/tests/resource_firebase_extensions_instance_test.go.erb:19]: Were these changes intended here (for `RandString` and `VcrTest`)?

File: mmv1/third_party/terraform/utils/provider_test.go.erb [hunk 2]
```diff
@@ -106,8 +106,8 @@ func TestAccProviderUserProjectOverride(t *testing.T) {
 	pid := "tf-test-" + RandString(t, 10)
 	topicName := "tf-test-topic-" + RandString(t, 10)
 
-	config := BootstrapConfig(t)
-	accessToken, err := setupProjectsAndGetAccessToken(org, billing, pid, "pubsub", config)
+	config := acctest.BootstrapConfig(t)
+	accessToken, err := acctest.SetupProjectsAndGetAccessToken(org, billing, pid, "pubsub", config)
```
Comment (roaks3) [inline mmv1/third_party/terraform/utils/provider_test.go.erb:110]: Same question here

**Review by roaks3 — APPROVED** [review]
Approving because it seems ok, but there were a couple lines that seem like they might have slipped in, and I'm not sure they are strictly related here.

File: mmv1/third_party/terraform/utils/provider_test.go.erb (hunk 2 above)
Comment (zli82016) [reply mmv1/third_party/terraform/utils/provider_test.go.erb:110]: Thanks for catching it. These changes should not in this PR.

File: mmv1/third_party/terraform/tests/resource_firebase_extensions_instance_test.go.erb (hunk 1 above)
Comment (zli82016) [reply mmv1/third_party/terraform/tests/resource_firebase_extensions_instance_test.go.erb:19]: Thanks for catching it. These changes should not in this PR.
